<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowDemo</class>
 <widget class="QMainWindow" name="ShowDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShowDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Showlabel">
      <property name="minimumSize">
       <size>
        <width>768</width>
        <height>576</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="InitDraw">
        <property name="text">
         <string>InitDraw</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ResizeWin">
        <property name="text">
         <string>ResizeWin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReleaseDraw">
        <property name="text">
         <string>ReleaseDraw</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>119</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="LoadImage">
        <property name="text">
         <string>LoadImage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="previewLB">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>144</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DrawWindow">
        <property name="text">
         <string>DrawWindow</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="openViedoFile">
        <property name="text">
         <string>OpenVideoFile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filePath">
        <property name="text">
         <string>G:\demovideo\test.h264</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="playbtn">
        <property name="text">
         <string>play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pausebtn">
        <property name="text">
         <string>pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopbtn">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="ImagePath">
      <property name="text">
       <string>G:\ShowDemo\bin\ori_vctest22.bmp</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="ThreadDraw">
      <property name="text">
       <string>ThreadDraw</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
